.travis.yml

```diff
@@ -12,6 +12,9 @@ python:
 service:
 - docker
 install:
+- python -m pip install --upgrade pip
+- python -m pip install --upgrade setuptools>=41.0.0
+- python -m pip install --upgrade tensorflow==2.0.0
```

Comment:
`tensorflow==2.0.0` will be installed by `step.py` via `python setup.py -q install`. Maybe remove it?

Reply:
I've tried what you say, but it's not viable. I think the `pip` and `setuptools` version in travis  is not up-to-date